lib/_debug_agent.js

```diff
@@ -81,6 +85,12 @@ Agent.prototype.notifyWait = function notifyWait() {
   this.first = false;
 };
 
+Agent.prototype.destroyAllClients = function destroyAllClients() {
+  this.clients.forEach(function(client) {
+    client.destroy();
```

Comment:
I will dig into this. I used destroy, because it was already used in the close event.
Maybe this.socket.end() would be sufficient here.